packages/analytics/analytics-geo-map/src/components/AnalyticsGeoMap.vue

```diff
@@ -0,0 +1,244 @@
+<template>
+  <div
+    id="mapContainer"
```

Comment:
issue: you’re using a Vue ref for an ID attribute. 

I would think you’d want to exclude ID in case there could be multiple instances on the same page?

Reply:
Whether `id` and/or `data-testid` - we should probably allow the consuming app to be in charge of generating unique ids

Reply:
Edit: well it’s actually a string, but it’s confusing because it matches the ref name 😅

Reply:
maplibre requires an id. The `ref` was not necessary

Reply:
Then the I’d should be required by the host app. If it’s built-in to the component you could end up with multiple on the same page